src/com/gh4a/adapter/IssueEventAdapter.java

```diff
@@ -156,6 +159,12 @@ protected Intent getIntent() {
             holder.tvDesc.setTag(null);
             holder.tvDesc.setText(formatEvent(event.event, event.getUser(),
                     holder.tvExtra.getTypefaceValue()));
+
+            Drawable issueEventIcon = getIssueEventIcon(event.event);
+            if (issueEventIcon != null) {
+                holder.ivIssueEvent.setVisibility(View.VISIBLE);
+                holder.ivIssueEvent.setImageDrawable(issueEventIcon);
+            }
```

Comment:
I know you noticed it already, just stating it explicitly:

```
} else {
    holder.ivIssueEvent.setVisibility(View.GONE);
}
```

is missing here.